### DISCUSSION TITLE
need a help

### DISCUSSION THREAD

**Discussion by omnia487** [securefederatedai/openfederatedlearning#303] [Q&A] [answered]
can you help me to use openfl with classification algorithms?

**Comment by alexey-gruzdev** [comment, marked as answer]
@omnia487 thanks for the interest in OpenFL. You could use [TinyImageNet tutorial ](https://github.com/intel/openfl/tree/develop/openfl-tutorials/interactive_api/PyTorch_TinyImageNet) as a reference point. Instructions are in general the same as for any other Interactive API tutorial, like [TensorFlow MNIST](https://github.com/intel/openfl/tree/develop/openfl-tutorials/interactive_api/Tensorflow_MNIST), so you could apply them by analogy.
